### PR TITLE
feature/vars_file

### PR DESCRIPTION
**Pull request by natemellendorf**
Adding support to pass in a var-file

### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### CODE REVIEW

**Review by murraybutler — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -146,7 +146,6 @@ workflows:
           name: dev-promote-patch
           orb-name: netgaintechnology/build-orb
           ssh-fingerprints: 9d:71:05:ff:1f:30:b2:c4:0b:9e:27:a9:09:d8:d0:c1
-          cleanup-tags: true
```
Comment (Stromweld) [inline .circleci/config.yml:149]: these are needed to cleanup the tags circleci creates to trigger the integration workflow.

**Review by Stromweld — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (natemellendorf) [reply .circleci/config.yml:149]: I’m looking into this now (via mobile) to try and see what changed / caused this to break.

Comment (natemellendorf) [reply .circleci/config.yml:149]: Found it. Below is the PR where this was merged into prod by CircleCI.
https://github.com/CircleCI-Public/orb-tools-orb/pull/64
Notes from RP:

> cleanup-tags isn't unused. it's a boolean for whether or not to do so. the cleanup-tags-pattern param then allows you to specify which tags to cleanup (if cleanup-tags is true). however, it is unused in the dev-promote-prod job in particular, where tags are always cleaned up & the user specifies only which tags should be removed. thank you !

**Review by Stromweld — APPROVED** [review]

**Pull request merged**